files/en-us/web/accessibility/understanding_colors_and_luminance/index.html

```diff
@@ -2,289 +2,502 @@
+
+<p>Flashing at rates higher than 3 per second over a screen area greater than about 300px square has been shown to induce photic seizures in some people, most likely in children and adolescents. It has also been found that certain very regular, high contrast patterns such as parallel white and black stripes, can also induce seizures. Some studies have indicated that very deep saturated red (with wavelengths of 660nm-700nm+) may exacerbate the problem, if these deep reds flash between a saturated and unsaturated state, no matter the luminance.</p>
+
+<p>The Epilepsy Foundation of America researched photic-related seizures thoroughly in this publication <a href="https://www.researchgate.net/publication/7615895_Photic-_and_Pattern-induced_Seizures_A_Review_for_the_Epilepsy_Foundation_of_America_Working_Group">Photic- and Pattern-induced Seizures A Review</a>. The study resulted in several important guidelines, a few key guidelines are:</p>
+
+<p><strong>Single, double, or triple flashes in one second are acceptable, but a sequence of flashes is not recommended when both of the following occur:</strong></p>
+
+<ol>
+  <li>More than three flashes within any one second period.</li>
+  <li>From the minimal expected viewing distance, the total area of concurrent flashes subtends at the eye a solid angle of >0.006 steradians (approximately 300px square).</li>
+</ol>
+
+<p><strong>When the light–dark stripes of any pattern are larger than 300px square (approximately) and the luminance of the lightest stripe is more than 50 cd/m², the pattern should display no more than:</strong></p>
+
+<ol>
+  <li>Five light–dark pairs of stripes, if the stripes change direction, oscillate, flash, or reverse in contrast.</li>
+  <li>Eight light–dark pairs of stripes, if the pattern is unchanging or continuously and smoothly drifting in one direction.</li>
+</ol>
+
+<p>The consensus recommendations are in this brief paper, <a href="https://www.epilepsy.com/sites/core/files/atoms/files/Epilepsia%20vol%2046%20issue%209%20Photosensitivity.pdf">Photic- and Pattern-induced Seizures: Expert Consensus of the Epilepsy Foundation of America</a>. Some additional insights are available in this UK paper covering <a href="https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.106.9473&rep=rep1&type=pdf">guidelines for preventing seizures.</a>.</p>
+
+<hr>
+
+<h2 id="Modeling_Light_Color_and_Vision">Modeling Light Color and Vision</h2>
+
+<p>To help us in our work, be it content design or determining accessibility, or calibrating a monitor, we need a way to model and predict the behavior and perception of light and color. These models are built with basic math, and are specific to the given task. A colorimetric model is one that is involved in measuring light, an appearance model is designed to predict how we will perceive the light as measured.</p>
+<h3 id="Calculating_Relative_Luminance">Calculating Relative Luminance</h3>
```

Comment:
```suggestion
<p>To help us in our work, be it content design or determining accessibility, or calibrating a monitor, we need a way to model and predict the behavior and perception of light and color. These models are built with basic math, and are specific to the given task. A colorimetric model is one that is involved in measuring light, an appearance model is designed to predict how we will perceive the light as measured.</p>

<h3 id="Calculating_Relative_Luminance">Calculating Relative Luminance</h3>
```

Inserting a newline between the paragraph and heading.